exercise-solutions/connected-mailbox/simple_db/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+authors = ["Ferrous Systems"]
+edition = "2021"
```

Comment:
I added `edition = "2021"` to this example. We should do this to all our examples. The default edition is `2015` and in that edition you have to use `extern crate foo;` to use some crate called `foo`, which is annoying.